internal/condition/deployment_config_condition.go

```diff
@@ -1,17 +1,18 @@
 package condition
 
 import (
-	"errors"
 	"fmt"
-	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 	"time"
+
+	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 )
 
-// DCCondition covers changes to Jenkins DeploymentConfigs
+// DeploymentConfigCondition covers changes to DeploymentConfigs
 type DeploymentConfigCondition struct {
 	idleAfter time.Duration
 }
 
+// NewDCCondition creates a new instance of DeploymentConfigCondition
 func NewDCCondition(idleAfter time.Duration) Condition {
```

Comment:
Wouldn't NewDeploymentConfigCondition be more appropriate?

Reply:
+1 Works for me